app/views/pages/_contributor_blurb.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="gridManyItem center">
+  <%= react_component('Modal', props: {
+    element: { 
+      component: 'Avatar', 
+      props: { 
+        src: ProfilePicture.normalize_url(blurb["image"]),
+        alt: t('pages.contribute.contributor_avatar_alt', {
+          name: blurb['name']
+        })
+      } 
+    },
+    body: <<~HEREDOC
+      <div class="contributorModalBody">
+        <div class="contributorModalBodyPhoto hideOnXSmall hideOnSmall">
+          <div class="avatar Avatar__avatar___3vcX_ Avatar__large___1dps1">
+            <img class="Avatar__image___2cXeU" src="#{ProfilePicture.normalize_url(blurb["image"])}">
+          </div>
+        </div>
+
+        <div class="contributorModalBodyCopy">
+          <div class="contributorModalHeader">
+            <div class="contributorModalBodyPhoto showOnSmall hideOnMedium hideOnLarge hideOnXLarge">
+              <div class="avatar Avatar__avatar___3vcX_ contributorModalAvatarMedium">
+                <img class="Avatar__image___2cXeU" src="#{ProfilePicture.normalize_url(blurb["image"])}">
+              </div>
+            </div>
+            <div>
+              <h4 class="contributorModalName">
+                #{blurb["name"]}
+              </h4>
+              <div class="contributorModalBodySocialLinks">
+                <i class="fa fa-location-arrow fa-inline"></i>#{blurb["location"]}
+                #{content_tag(:i, '', class: "#{get_icon_class(blurb["social"])} fa-inline")}
+                #{link_to(blurb["link_name"], blurb["link"], target: 'blank')}
+              </div>
```

Comment:
On smaller breakpoints, it would be great to separate out both location and social link onto their own lines, so perhaps wrapping each in a `<div>` would be good. You can then style them to be happy with each other on the same line as the screen gets larger.